packages/UnorderedList/UnorderedItem/UnorderedItem.jsx

```diff
@@ -1,31 +1,43 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import Text from '@tds/core-text'
-
 import safeRest from '../../../shared/utils/safeRest'
 import joinClassNames from '../../../shared/utils/joinClassNames'
 
 import styles from './UnorderedItem.modules.scss'
 import listStyles from '../../../shared/styles/List.modules.scss'
 
-const UnorderedItem = ({ listStyle, size, children, ...rest }) => (
+const UnorderedItem = ({ listStyle, itemStyle, size, children, ...rest }) => (
   <li
     {...safeRest(rest)}
-    className={joinClassNames(styles[listStyle], styles[size], listStyles.item)}
+    className={joinClassNames(styles[itemStyle || listStyle], styles[size], listStyles.item)}
   >
-    <Text size={size}>{children}</Text>
+    {children}
   </li>
 )
 
 UnorderedItem.propTypes = {
-  listStyle: PropTypes.oneOf(['circle', 'checkmark', 'x']),
+  /**
+   * The list's global bullet style. Can be overriden by listStyle.
+   * @ignore
+   */
+  listStyle: PropTypes.oneOf(['', 'circle', 'checkmark', 'x']),
```

Comment:
Having `''` as a prop value here is unintended. It should be `PropTypes.oneOf(['circle', 'checkmark', 'x']),`